invenio_jsonschemas/jsonresolver.py

```diff
@@ -41,3 +43,8 @@ def jsonresolver_loader(url_map):
         "{0}/<path:path>".format(current_app.config['JSONSCHEMAS_ENDPOINT']),
         endpoint=current_app.extensions['invenio-jsonschemas'].get_schema,
         host=current_app.config['JSONSCHEMAS_HOST']))
+
+
+loader = json_loader_factory(LocalProxy(lambda: jsonresolver.JSONResolver(
+    entry_point_group='invenio_records.jsonresolver'
```

Comment:
(a) maybe we should rename it or (b) we can create a new default with plugins: `JSONResolver(plugins=...)`